man.Rmd

```diff
@@ -177,9 +242,20 @@ When writing the title, it's useful to think about how it will appear in the ref
 When a user skims the index, how will they know which functions will solve their current problem?
 This requires thinking about what your functions have in common (which doesn't need to be repeated in every title) and what is unique to that function (which should be highlighted in the title).
 
-As an example, take the titles of some of the key functions in dplyr[^man-3]:
+When we wrote this chapter, we found the function titles for stringr to be somewhat disappointing.
```

Comment:
I think it's better to show the underwhelming titles first, then the good ones. End on a happy note.